### ISSUE TITLE
Adding Hide/Unhide button on list/table elements for integration/plugin/...

### ISSUE THREAD

**Issue by olijouve** [hacs/frontend#49]
**Is your feature request related to a problem? Please describe.**
Being able to Hide component is a very nice feature, it offers a clearest view by filtering out by status for what we can considerate as unappropriate elements(unapplyable country specifics...).
But having to enter in each integration to Hide from the drop down menu can be plainfull, especially because number of integration is quite huge now(and it's a good thing), but navigate back from element details page to listing page loose last scrolling position and return to the top...

**Describe the solution you'd like**
It would be nice to have an Hide action button/icon directly in the integration/plugin/... listing for each elements. Request would be played in background so we could keep staying in the listing page.

**Describe alternatives you've considered**
An alternative could also be, just like the "Clear New" link, to have a "Hide All" link, then by filtering by Name or Stars we could have an Unhide action button/icon directly in the integration/plugin/... listing for each elements.

**Comment by ludeeus** [comment]
So, in short, you want to be able to do mass operations from the overviews?
That an interesting idea, but I have absolutely no clue how do manage that.

**Comment by olijouve** [comment]
Yes it's the idea, but not necessarily by mass operations from the overviews like checkbox array / form submit.

I'd rather think of having an Hide bouton/link on each element in the overview that would just do the same individual and backgrounded Hide operation we can have in the menu of the component details page.

So if i want to Hide 10 components/plugins... i would have to Hide them one by one but directly from the overview page.

**Comment by ludeeus** [comment]
Adding buttons (clutter) to the overviews I want to avoid as much as possible.
Adding 1 button seems harmless, but then someone will have other buttons there as well, it will quickly be messy if I can add an edit/manage mode or something similar it's doable.

**Comment by Fredderic** [comment]
This is true…  I was just considering opening an issue about one or two of those buttons…

I want a checkbox in the bottom-right corner of each new and updatable thing, and a button to Update or Acknowledge them en-masse.  (Hiding stuff I don't think I'm ever going to be interested in might be nice, too…)

You could get away with a pop in right hand sidebar, showing the set of actions which can be applied to any selected cards (and perhaps highlight the relevant cards when you hover over an action).

Or something.

I just *really* want to be able to select a bunch of upgradable things to upgrade, or new things to acknoledge their newness.

**Comment by Petro31** [comment]
I think this could be done if we force the table on people.

Treat it like an email list.  Add a checkbox for selection.  Add shift + click, ctrl + click for multi selections.  Then add operations in a dropdown to perform.  I.E.  Hide, Update, etc.

**Comment by ludeeus** [comment]
I personally do not like the list view.
As a MVP it will only be added to the grid view.

**Comment by olijouve** [comment]
It will be good enough in grid view :)

**Comment by Petro31** [comment]
Just not sure how you'd implement a multi select in a grid view.  Possibly like windows select on desktop?  CTRL, SHIFT, and also make a box?
